mesh/profiles/views.py

```diff
@@ -77,3 +85,27 @@ def get_data(account_id, name, mapper):
             "status": "error",
             "message": "An account does not exist with this account ID."
         }, status=404)
+
+def post_data(account_id, name, request):
+    """
+    Handles POST requests when the client sends names/nicknames/prounouns to the back end
+    """
+    try: 
+        profile = Profile.objects.get(accountID = account_id)
+        data = request.POST[name]   
+
+        if (name == "userName"):
+            profile.userName = data 
+        elif (name == "preferredName"):
+            profile.preferredName = data
+        elif (name == "preferredPronouns"):
+            profile.preferredPronouns = data
+    
+        profile.save()
+        return JsonResponse({'message': f'{name} saved successfully'}, safe=False, status = 200)
```

Comment:
I think here, the `safe=False` parameter can be removed, as the `safe` parameter only needs to be set to false if we are sending objects that are not JSON serializable, but in this case, you are just returning a string in the message, so `safe = False` shouldn't be necessary (though I would still test it to see if I'm correct or not). 

Point being that I don't see why `safe = False` would be needed here but not on line 111, which is nearly identical to this one.

Reply:
Yeah, you are right. I removed it 👍.